### DISCUSSION TITLE
fido2 error

### DISCUSSION THREAD

**Discussion by quer0016** [JanssenProject/jans#7846] [Q&A] [answered]
Hello team, 
I have a problem with the operation of fido2 for the use of yubikey, in the main installation I did not consider the use of yubikey and I carried out the installation without this plugin, I have carried out the manual post-installation installation but now the service gives me an error and I can't find the reason.

**Comment by nynymike** [comment]
Can you just do a re-install?

**Reply by quer0016** [reply]
Hello Mike, Yes, in this case I can do it because it is a development environment, but in the case of a production environment, is it the only way to solve it?

**Reply by nynymike** [reply, marked as answer]
There is always a way to fix it. But hopefully by the time you get to prod, you'll know what you want to deploy the first time :-)
